### DISCUSSION TITLE
Proposition de Logo v0.1

### DISCUSSION THREAD

**Discussion by AdrienDSlone** [AdrienDSlone/slowTripPlanner#32] [3. UI/UX]
Je vous propose un logo qui ressemble à un escargot. L'escargot est constitué d'une multitude de balises gps (l'icone universel des cartes en ligne et des gps).

Qu'en pensez-vous ?
![20220609_logo](https://user-images.githubusercontent.com/50139099/172859390-7eeedbf2-8989-477f-a7d5-d68ef86969c0.png)

**Comment by AdrienDSlone** [comment]
La version 2 : 
![20220609_logo2](https://user-images.githubusercontent.com/50139099/172862931-68824409-7edd-47fa-8248-a4042bf63381.png)
:

**Comment by AdrienDSlone** [comment]
Je vous propose cette variante : 
![image](https://user-images.githubusercontent.com/50139099/172907092-d6b1b100-22b2-4849-b584-fc335aba874c.png)

et la version en négatif :
![image](https://user-images.githubusercontent.com/50139099/172907418-30fc2038-b7fc-4dbf-a4c6-3589c3278129.png)

**Comment by jdenizart** [comment]
![IMG_20220701_094210](https://user-images.githubusercontent.com/104148814/176883525-5cef829d-96fd-4293-b9d7-81a1b0d5e95b.jpg)
